packages/angular/projects/angular/src/components/radio-button/radio-button.component.scss

```diff
@@ -0,0 +1,10 @@
+@import '@devextreme/styles/src/radio-button/radio-button.scss';
```

Comment:
Question: Do I understand right? It is Angular-specific CSS rules because Angular renders additional DOM elements and common styles don't work.

Reply:
Eup, we need additional styles for additional Angular component DOM elements.